tests/queries/0_stateless/02677_hash_functions_on_null.sql

```diff
@@ -0,0 +1,15 @@
+select xxHash32(NULL);
+select xxHash64(NULL);
+
+DROP TABLE IF EXISTS test_hash_on_null;
+CREATE TABLE test_hash_on_null (a Array(Nullable(Int64))) ENGINE = Memory;
+insert into test_hash_on_null values (NULL) ([NULL, NULL]);
```

Comment:
The column type is `Array(Nullable(T))`. What is the actually stored value if `NULL` is inserted? (also wondering, how that is even possible?)